resource_management/ResourceRequest[] requests
---
uint32 grantID
resource_management/ResourceGrant[] grants
